### ISSUE TITLE
x-info-box-action not rendering in demos?

### ISSUE THREAD

**Issue by mkontula** [infusion-code/angular-maps#79]
I was previewing the lib for our project and noticed that in this demo: https://google-basic-map-with-data-bound-markers.stackblitz.io/ x-info-box-action element does not seem to render at all. It also does not produce any errors or other effects. Is it just me or is there something wrong with the demo/markup?

**Comment by thor-schueler** [comment]
Good point. ```x-info-box-action``` is currently only working in the bing implementation. That is because we are relying on the bing capabilities to add actions to the info box (the x-info-box-action actually does not really render anything, we are just using it to enumerate and add to the infobox options in the implementation of the bing-infobox.service.ts). Google does not have a similar capability. 

Currently, in google, you would have to implement this in a custom way using custom HTML/ng inside the <x-info-box> tag. Such as 

                <x-info-box [DisableAutoPan]="true" [Title]="'My InfoBox ' + i.toString()">
                    Ping...
                    You should be able to use any markup or ng directives in here.  
                </x-info-box>

It would be nice to change the implementation in google-infobox.service.ts to dynamically generate and render the appropriate HMTL.
